.github/workflows/docker-build.yaml

```diff
@@ -23,19 +23,12 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v2
 
-      - name: Login to Docker Hub
-        uses: docker/login-action@v3
-        if: ${{ github.ref == 'refs/heads/master' }}
-        with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
-
       - name: Build
         uses: docker/build-push-action@v5
         with:
           context: .
           outputs: type=docker,dest=/tmp/docker.tar
-          tags: ${{ secrets.DOCKERHUB_USERNAME }}/bancho.py:latest
+          tags: bancho.py:${{ github.sha }}

```

Comment:
was considering using ${{ github.repository_owner }} or the event owner (pr creator) but that that is a bit annoying to set up due to needing to be all lowercase. if viewed as something necessary it can be done